include/revng/Pipeline/LLVMGlobalKindBase.h

```diff
@@ -54,6 +54,19 @@ class LLVMGlobalKindBase : public KindForContainer<LLVMContainer> {
     return false;
   }
 
+  llvm::Error
+  verify(const ContainerBase &Container, const Target &T) const override {
+    if (const auto *Casted = llvm::dyn_cast<LLVMContainer>(&Container);
+        Casted != nullptr)
```

Comment:
You can drop `Casted != nullptr`

Reply:
why? Container is a argument that may not be a LLVMContainer

Reply:
```c++
if (const auto *Casted = llvm::dyn_cast<LLVMContainer>(&Container);
        Casted != nullptr)
```
is the same as
```c++
if (const auto *Casted = llvm::dyn_cast<LLVMContainer>(&Container))
```
isn't it?

Reply:
ahh, right, it just look more explicit the current way, but ok, make sense